README.md

```diff
@@ -197,9 +201,13 @@ export INDEXER_BRANCH="develop"
 export INDEXER_SHA=""
 export INDEXER_DISABLED=""
 export INDEXER_ENABLE_ALL_PARAMETERS="false"
+export CONDUIT_URL="https://github.com/algorand/conduit"
+export CONDUIT_BRANCH="master"
+export CONDUIT_SHA=""
+export CONDUIT_DISABLED=""
 ```
 
-Indexer is always built from source since it can be done quickly. For most configurations, algod will be installed using our standard release channels, but building from source is also available by setting the git URL, Branch and optionally a specific SHA commit hash.
+Indexer and Conduit are always built from source since it can be done quickly. For most configurations, algod will be installed using our standard release channels, but building from source is also available by setting the git URL, Branch and optionally a specific SHA commit hash.
```

Comment:
```suggestion
Indexer and Conduit are always built from source since these can be done quickly. For most configurations, algod will be installed using our standard release channels, but building from source is also available by setting the git URL, branch and optionally a specific SHA commit hash.
```
pluralizing first sentence and un-capping **Branch**